src/components/Pages/Search/searchResult.js

```diff
@@ -2,10 +2,9 @@ import React from 'react';
 import moment from 'moment-timezone';
```

Comment:
About the searchResult page. We're now getting all of this data straight from our search_output function in Joplin. So our API can now be more precise about which pieces of data it wants to return. That's why you'll see a bunch of these values be changed. For example, instead of `janisUrls && janisUrls[0]`, we can now just use the `url` our site_search API returned.